examples/intervention_approvals/intervention.py

```diff
@@ -0,0 +1,290 @@
+from rich.console import Group, RenderableType, Console
+
+from inspect_ai.model._chat_message import ChatMessageAssistant
+console = Console()
```

Comment:
Rich has a global default console which you can access with `rich.get_console()`. That's the one Inspect uses so that's the one you should use here.